src/refmt_impl.ml

```diff
@@ -35,6 +35,22 @@ let reasonBinaryParser use_stdin filename =
   let (magic_number, filename, ast, comments, parsedAsML, parsedAsInterface) = input_value chan in
   ((ast, comments), parsedAsML, parsedAsInterface)
 
+let usage =
```

Comment:
If you want to change to the sweet multiline form, I'll merge this immediately after.